CONTRIBUTING.md

```diff
@@ -63,15 +63,15 @@ drop the `-daemon` switch, and start the CLI in a second console.
 
 #### Building a released version
 
-The source for standalone ksqlDB versions is tagged with the pattern vN.NN.N-ksqldb. For
-example, the tag for the 0.18.0 standalone release is v0.18.0-ksqldb.
+The source for standalone ksqlDB versions is tagged with the pattern vN.NN.N. For
+example, the tag for the 0.18.0 standalone release is v0.18.0.
```

Comment:
Can we use whatever the latest version is at the moment, i.e. `0.28.2`? `0.18.0` really begs for upgrade :)

Reply:
Updated